src/clusterd/src/bin/clusterd.rs

```diff
@@ -260,6 +260,14 @@ async fn run(args: Args) -> Result<(), anyhow::Error> {
     let mut persist_cfg =
         PersistConfig::new(&BUILD_INFO, SYSTEM_TIME.clone(), mz_dyncfgs::all_dyncfgs());
     persist_cfg.is_cc_active = args.is_cluster_size_v2;
+
+    // Disable PubSub by default in `clusterd`. If PubSub starts enabled in
```

Comment:
I think I follow the "If PubSub starts enabled" part, but not sure I understand the second part about ensuring the connection parameters are synced before the connection. Isn't that what `configs_synced_once` does?

Reply:
Oops, sorry, yes! This bit is not necessary with the `configs_synced_once` bool. It's a vestige of the original approach.